test/e2e/traceflow_test.go

```diff
@@ -2012,6 +2016,8 @@ func testTraceflowExternalIP(t *testing.T, data *TestData) {
 	nodeIP := nodeIP(nodeIdx)
 	podNames, _, cleanupFn := createTestAgnhostPods(t, data, 1, data.testNamespace, node)
 	defer cleanupFn()
+	// Give a little time for Nodes to install OVS flows.
```

Comment:
Maybe add some reason in the comment to explain why more time is needed with containerd?

Reply:
reminder

Reply:
As we discussed, creating containerd container calls a different endpoint workflow,  which may cause different delay compared to docker.